lib/ship.js

```diff
@@ -1,8 +1,14 @@
 var canvas = document.getElementById('game');
+var largeAsteroidRadius = 20;
```

Comment:
It was Nate's suggestion. To make a smooth transition - rather than having the asteroid blink to the opposite side - let the asteroid slide all the way off the visible area before warping it.